crates/block-producer/src/lib.rs

```diff
@@ -16,21 +16,24 @@ pub mod server;
 // CONSTANTS
 // =================================================================================================
 
-/// The name of the block producer component
+/// The name of the block producer component.
 pub const COMPONENT: &str = "miden-block-producer";
 
-/// The number of transactions per batch
+/// The number of transactions per batch.
 const SERVER_MAX_TXS_PER_BATCH: usize = 2;
 
-/// The frequency at which blocks are produced
+/// The frequency at which blocks are produced.
 const SERVER_BLOCK_FREQUENCY: Duration = Duration::from_secs(5);
 
-/// The frequency at which batches are built
+/// The frequency at which batches are built.
 const SERVER_BUILD_BATCH_FREQUENCY: Duration = Duration::from_secs(2);
 
-/// Maximum number of batches per block
+/// Maximum number of batches per block.
 const SERVER_MAX_BATCHES_PER_BLOCK: usize = 4;
 
+/// Size of the batch building worker pool.
+const SERVER_BATCH_BUILDERS: NonZeroUsize = NonZeroUsize::new(2).unwrap();
```

Comment:
nit: I would name this something like `NUM_SERVER_BATCH_BUILDERS` to make it clear that this is a number.

Reply:
It is, but that's how we usually name "countable" variables in most other places.